### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by dflorent**
Hi ! Thanks for your work. I'm using your project right now. I propose a translation in French.

### CODE REVIEW

File: languages/papi-fr_FR.po [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+msgstr ""
+"Sélectionnez le type de page à créer à partir de la liste ou la recherche"
+
+msgid "The Standard Page"
+msgstr "La page standard"
+
+msgid "Add New2"
+msgstr "Ajouter nouveau"
+
+msgid "Normal page"
```
Comment (frozzare) [inline languages/papi-fr_FR.po]: "Normal Page" should removed

File: languages/papi-fr_FR.po [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+msgstr "Recherche d’adresse…"
+
+msgid "Select the type of page to create from the list or search"
+msgstr ""
+"Sélectionnez le type de page à créer à partir de la liste ou la recherche"
+
+msgid "The Standard Page"
+msgstr "La page standard"
+
+msgid "Add New2"
```
Comment (frozzare) [inline languages/papi-fr_FR.po]: "Add New2" should be removed

File: languages/papi-fr_FR.po [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+msgstr "Impossible de déterminer l’adresse à cet endroit."
+
+msgid "Search for address..."
+msgstr "Recherche d’adresse…"
+
+msgid "Select the type of page to create from the list or search"
+msgstr ""
+"Sélectionnez le type de page à créer à partir de la liste ou la recherche"
+
+msgid "The Standard Page"
```
Comment (frozzare) [inline languages/papi-fr_FR.po]: "The Standard Page" should be removed

File: languages/papi-fr_FR.po [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+msgid "Standard page"
+msgstr "Page standard"
+
+msgid "Cannot determine address at this location."
+msgstr "Impossible de déterminer l’adresse à cet endroit."
+
+msgid "Search for address..."
+msgstr "Recherche d’adresse…"
+
+msgid "Select the type of page to create from the list or search"
```
Comment (frozzare) [inline languages/papi-fr_FR.po]: "Select the type of page to create from the list or search" should be removed

File: languages/papi-fr_FR.po [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+msgid "(required field)"
+msgstr "(champ obligatoire)"
+
+msgid "Standard page"
+msgstr "Page standard"
+
+msgid "Cannot determine address at this location."
+msgstr "Impossible de déterminer l’adresse à cet endroit."
+
+msgid "Search for address..."
```
Comment (frozzare) [inline languages/papi-fr_FR.po]: "Search for address..." should be removed

File: languages/papi-fr_FR.po [hunk 6]
```diff
@@ -0,0 +1,263 @@
...
+msgid "No meta boxes exists."
+msgstr "Aucune metaboxes existante."
+
+msgid "(required field)"
+msgstr "(champ obligatoire)"
+
+msgid "Standard page"
+msgstr "Page standard"
+
+msgid "Cannot determine address at this location."
```
Comment (frozzare) [inline languages/papi-fr_FR.po]: "Cannot determine address at this location." should be removed

File: languages/papi-fr_FR.po [hunk 7]
```diff
@@ -0,0 +1,263 @@
...
+msgstr "Retour à la liste"
+
+msgid "No meta boxes exists."
+msgstr "Aucune metaboxes existante."
+
+msgid "(required field)"
+msgstr "(champ obligatoire)"
+
+msgid "Standard page"
+msgstr "Page standard"
```
Comment (frozzare) [inline languages/papi-fr_FR.po]: "Standard page" should be removed

File: languages/papi-fr_FR.po [hunk 8]
```diff
@@ -0,0 +1,263 @@
...
+msgid "Items"
+msgstr "Items"
+
+msgid "Overview of page types"
+msgstr "Toutes les types de page"
+
+msgid "Just the normal WordPress page"
+msgstr "Juste la page WordPress normale"
+
+msgid "Select post"
```
Comment (frozzare) [inline languages/papi-fr_FR.po]: "Select post" should be removed

File: languages/papi-fr_FR.po [hunk 9]
```diff
@@ -0,0 +1,263 @@
...
+"WordPress 4.0 et supérieur est requis pour exécuter Papi ! Le plugin s’est "
+"maintenant désactivé."
+
+msgid "Items"
+msgstr "Items"
+
+msgid "Overview of page types"
+msgstr "Toutes les types de page"
+
+msgid "Just the normal WordPress page"
```
Comment (frozzare) [inline languages/papi-fr_FR.po]: "Just the normal WordPress page" should be removed

File: languages/papi-fr_FR.po [hunk 10]
```diff
@@ -0,0 +1,263 @@
...
+"WordPress 4.0 and higher required to run Papi! The plugin has now disabled "
+"itself."
+msgstr ""
+"WordPress 4.0 et supérieur est requis pour exécuter Papi ! Le plugin s’est "
+"maintenant désactivé."
+
+msgid "Items"
+msgstr "Items"
+
+msgid "Overview of page types"
```
Comment (frozzare) [inline languages/papi-fr_FR.po]: "Overview of page types" should be removed

File: languages/papi-fr_FR.po [hunk 11]
```diff
@@ -0,0 +1,263 @@
...
+
+#: ../src/papi-loader.php:163
+msgid ""
+"WordPress 4.0 and higher required to run Papi! The plugin has now disabled "
+"itself."
+msgstr ""
+"WordPress 4.0 et supérieur est requis pour exécuter Papi ! Le plugin s’est "
+"maintenant désactivé."
+
+msgid "Items"
```
Comment (frozzare) [inline languages/papi-fr_FR.po]: "Items" should be removed

File: languages/papi-fr_FR.po [hunk 12]
```diff
@@ -0,0 +1,263 @@
...
+"Last-Translator: Fredrik Forsmo <fredrik.forsmo@gmail.com>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.1\n"
+"X-Poedit-Basepath: .\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"X-Poedit-KeywordsList: _e;__\n"
```
Comment (frozzare) [inline languages/papi-fr_FR.po]: I have updated the language file and this should be.

```
"X-Poedit-KeywordsList: _e;__;esc_html__;esc_html_e\n"
```

**Comment by frozzare** [comment]
I have updated the translation file since there was old stuff left and added `esc_html__` and `esc_html_e` to the keyword list.

**Comment by dflorent** [comment]
I have updated. I let you check out.

**Comment by frozzare** [comment]
Looks better! Thanks for your pull request!

**Pull request merged**